docs/fsharp/getting-started-netcore.md

```diff
@@ -0,0 +1,313 @@
+
+## Prerequisites
+
+To begin, you must install the [.NET Core 1.0 SDK Preview 2](https://www.microsoft.com/net/core).
+
+This article uses the command line and [Visual Studio Code](https://code.visualstudio.com) as the text editor.  You can use any editor you like.
+
+## Building a Simple Multi-project Solution
+
+1. Open up a Command Line/Terminal.
+2. Create a new directory named `FSNetCore`.  Open Visual Studio code or your preferred editor inside this directory. 
+3. Under `FSNetCore`, create `src` and `test` directories.
+4. Under `FSNetCore`, create a new file called `global.json`.  It should have this as its contents:
+
+```json
+{
+    "projects":[ "src", "test" ]
+}
+```
+
+Your solutions structure should now look like this:
+
+```
+FSNetCore/
+|---src/
+|---test/
+|---global.json
+```
+
+### Writing a Class library
+
+1. Create a `Library` folder under `FSNetCore/src`.
+2. In the command line, execute `dotnet new -l F#` in `FSNetCore/src/Library`.
+3. Remove the `NuGet.Config` file.
+4. Rename `Program.fs` to `Lib.fs`.
+5. Open the `project.json` file and remove the `emitEntryPoint` entry from `buildOptions`.
+6. Under `buildOptions/compile/includeFiles`, replace `Program.fs` with `Lib.fs`.
+7. Remove the global `dependencies` section.
+8. Under `frameworks`, change `netcoreapp1.0` to `netstandard1.6`.
+9. Under `frameworks/netstandard1.6`, remove the `imports` section.
+10. Under `frameworks/netstandard1.6/dependencies`, replace the `Microsoft.NETCore.App` package with `"NETStandard.Library":"1.6.0"`.  Add `"Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629"` and `"Newtonsoft.Json": "9.0.1-beta1"`.
+11. Open `Lib.fs` and change the contents to the following code:
+    ```fsharp
+    module Library
+
+    open Newtonsoft.Json
+
+    let getJsonNetJson value = 
+        sprintf "I used to be %s but now I'm %A!" <|| (value,  JsonConvert.SerializeObject(value))
+    ```
+12. Run `dotnet restore` and `dotnet build`.  These should succeed.
+
+Your `project.json` file should look like this:
+
+```json
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "compilerName": "fsc",
+    "compile": {
+      "includeFiles": [
+        "Lib.fs"
+      ]
+    }
+  },
+  "tools": {
+    "dotnet-compile-fsc": {
+      "version": "1.0.0-preview2-*",
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win81",
+        "netstandard1.3"
+      ]
+    }
+  },
+  "frameworks": {
+    "netstandard1.6": {
+      "dependencies": {
+        "NETStandard.Library":"1.6.0",
+        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
+        "Newtonsoft.Json": "9.0.1-beta1"    
+      }
+    }
+  }
+}
+```
+
+And your `Lib.fs` file should look like this:
+
+```fsharp
+module Library
+
+open Newtonsoft.Json
+
+let getJsonNetJson value = 
+    sprintf "I used to be %s but now I'm %A!" <|| (value,  JsonConvert.SerializeObject(value))
+```
```

Comment:
Why repeat how Lib.fs should look like here? Step 11 above contains exactly the same contents.